MovieTime/MovieTime/Core/MTCellProtocol.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  MTCellProtocol.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 09/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import UIKit
+
+protocol MTCellProtocol: class {
```

Comment:
try to stay away from the practice of adding initials to the name of entities. You'll realize that we tend to use them when we can't give a better name. A good name is the one that declares the entity's responsibility. When you say it's an MTSomething we have no idea what's special about that something ;)